scripts/ci/libraries/_kind.sh

```diff
@@ -195,6 +197,16 @@ function kind::perform_kind_cluster_operation() {
             echo "Entering an interactive shell for kubernetes testing"
             echo
             "${AIRFLOW_SOURCES}/scripts/ci/kubernetes/ci_run_kubernetes_tests.sh" "-i"
+        elif [[ ${OPERATION} == "k9s" ]]; then
+            echo
+            echo "Starting k9s CLI"
+            echo
+            export TERM=xterm-256color
+            export EDITOR=vim
```

Comment:
Note - we are runnig k9s as docker image so we have to have editor that is installed there :). I picked vim :)